functions-and-operators/string-functions.md

```diff
@@ -1396,11 +1378,60 @@ Return a string containing hex representation of a number.
 
 ### [`UPPER()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_upper)
 
-Convert to uppercase.
+The `UPPER()` function is used to convert a string to uppercase letters. This function is equivalent to the `UCASE()` function.
+
+> **Note:**
+>
+> When the string is `NULL`, the `UPPER()` function returns `NULL`.
+
+Example:
+
+```sql
+SELECT upper('bigdata') AS result_upper, upper('null') AS result_null;
```

Comment:
The example is a little tricky here. The Note "When the string is `NULL`, the `UPPER()` function returns `NULL`", actually mean, upper(null) instead of upper('null'), the latter one is just a normal string.
Better changed to SELECT upper('bigdata') AS result_upper, upper(null) AS result_null;

Reply:
Got it.